NewForce-Cohort-6/tabloid-cli-coven-of-the-lunar-owls
# Ticket #16 - List Posts

# Description

This PR is for [Ticket #16](https://trello.com/c/Ybfzry4t), which should allow the user to navigate to Post Management from the Main Menu and then return a list of posts from option one of the Post Menu. This has been tested in the CLI and displays the expected result from the database, which is the title and URL of a given post.

# Testing Instructions

1. git fetch --all 
2. git checkout ac-listPosts to switch to my branch. 
If you don't see the most recent version of the files, try git pull origin ac-listPosts.
3. Execute TabloidCLI from inside of Visual Studio
4.  Select option 4, Post Management, from Main Menu.
5.  Select option 1, List Posts, from Post Menu.

# Checklist:
